src/networkmessage.h

```diff
@@ -149,28 +146,31 @@ class NetworkMessage
 		}
 
 		uint8_t* getBodyBuffer() {
-			position = 2;
+			info.position = 2;
 			return buffer + HEADER_LENGTH;
 		}
 
 	protected:
 		inline bool canAdd(size_t size) const {
-			return (size + position) < MAX_BODY_LENGTH;
+			return (size + info.position) < MAX_BODY_LENGTH;
 		}
 
 		inline bool canRead(int32_t size) {
-			if ((position + size) > (length + 8) || size >= (NETWORKMESSAGE_MAXSIZE - position)) {
-				overrun = true;
+			if ((info.position + size) > (info.length + 8) || size >= (NETWORKMESSAGE_MAXSIZE - info.position)) {
+				info.overrun = true;
 				return false;
 			}
 			return true;
 		}
 
-		MsgSize_t length;
-		MsgSize_t position;
-		bool overrun;
+		struct NetworkMessageInfo {
+			MsgSize_t length = 0;
+			MsgSize_t position = INITIAL_BUFFER_POSITION;
+			bool overrun = false;
+		};
 
-		uint8_t buffer[NETWORKMESSAGE_MAXSIZE];
+		NetworkMessageInfo info;
+		uint8_t buffer[NETWORKMESSAGE_MAXSIZE] = {};
```

Comment:
Actually, for the constexpr specifier to work it needs to initialize all attributes